packages/governance/src/cip36.ts

```diff
@@ -17,18 +17,18 @@ export enum MetadataLabel {
 export const SIG_MAP_KEY = 1n;
 
 /**
- * CIP-36 voting key derivation path constants
+ * CIP-36 vote key derivation path constants
  */
-export enum VotingKeyDerivationPath {
+export enum CIP36VoteKeyDerivationPath {
   PURPOSE = keyManagementUtil.harden(1694),
   COIN_TYPE = keyManagementUtil.harden(1815)
 }
 
 /**
- * Voting power delegation to a specified voting key
+ * Voting power delegation to a specified vote key
  */
-export interface GovernanceKeyDelegation {
-  votingKey: Cardano.Ed25519PublicKey;
+export interface VoteKeyDelegation {
+  CIP36VoteKey: Cardano.Ed25519PublicKey;
```

Comment:
Let's stick to this naming convention
```suggestion
  cip36VoteKey: Cardano.Ed25519PublicKey;
```

Reply:
Good idea, I will make this change and amend the commit message